rootfs/etc/cont-init.d/08-svc-snmptrapd.sh

```diff
@@ -20,6 +20,6 @@ mkdir -p /etc/services.d/snmptrapd
 cat > /etc/services.d/snmptrapd/run <<EOL
 #!/usr/bin/execlineb -P
 with-contenv
-/usr/sbin/snmptrapd -f -m IF-MIB -M /opt/librenms/mibs:/opt/librenms/mibs/cisco
+/usr/sbin/snmptrapd -f -m ALL -M /opt/librenms/mibs:/opt/librenms/mibs/cisco
```

Comment:
LibreNMS has a lot of MIBs, this might break snmptrapd.
You kind of need away for users to specify this.
It would be cool if there was a way to configure this from LibreNMS, but I have never figured out a way to update snmptrapd at runtime in an os agnostic way.

Reply:
I don't see a issue with the current way snmptrapd loads mib's, and it works, so I'd suggest if you have the need to configure this you work on top of these patches to add that.

Reply:
There is an issue with current approach as it loads only partial MIBs, please note the MIB searching is not recursive, so, you need to specifiy all possible sub-directories as well. This is why it's better to amke it configurable.

Reply:
I know that mib parsing isn't recursive, but this loads just fine for now, and enough for MVP. Of course it can be improved, but don't let that block this PR.